server/views/pages/create/additionalConditions.njk

```diff
@@ -20,98 +20,42 @@
         <div class="govuk-grid-row">
             <div class="govuk-grid-column-full">
 
-                {{ govukCheckboxes({
-                    name: "additional-conditions",
-                    fieldset: {
-                        legend: {
-                            text: "Residence at a specific place",
-                            isPageHeading: true,
-                            classes: "govuk-fieldset__legend--l"
-                        },
-                        classes: "govuk-!-margin-top-6"
-                    },
-                    items: [
-                        {
-                            value: "residence-condition-id-0",
-                            text: "You must reside within [INSERT REGION] while of no fixed abode, unless otherwise approved by your supervising officer.",
-                            conditional: {
-                                html: govukInput({
-                                    name: "supplementary-data[residence-condition-id-0][probation-region]",
-                                    classes: "govuk-!-width-one-third",
-                                    label: {
-                                        text: "Enter probation region"
-                                    }
+                {% for group in additionalConditions %}
+                    {% set items = [] %}
+                    {% for condition in group.conditions %}
+                        {% set conditionalHtml = {} %}
+                        {% if condition.inputTemplate %}
+                            {% set conditionalHtml = {
+                                html: formBuilder({
+                                    id: condition.id,
+                                    inputTemplate: condition.inputTemplate,
+                                    errorMessages: validationErrors,
```

Comment:
This is quite neat. I can see why form-builder looks a reasonable choice here. Lets try run with this, and we could change if we see problems down the line.